.github/workflows/ci.yml

```diff
@@ -190,8 +190,15 @@ jobs:
           version: ${{ env.DOCKER_BUILDX_VERSION }}
           install: true
 
+      - name: Login to ghcr.io
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
```

Comment:
this is following what https://github.com/crossplane-contrib/provider-workflows/blob/main/.github/workflows/publish-provider-non-family.yml#L74 does